chapter_builders-guide/lazy-init.md

```diff
@@ -74,11 +74,30 @@ net = tf.keras.models.Sequential([
 ])
 ```
 
+```{.python .input}
+%%tab jax
+from d2l import jax as d2l
+from flax import linen as nn
+import jax
+from jax import numpy as jnp
+
+net = nn.Sequential([nn.Dense(256), nn.relu, nn.Dense(10)])
+```
+
 At this point, the network cannot possibly know
 the dimensions of the input layer's weights
 because the input dimension remains unknown.
+
+:begin_tab:`mxnet, pytorch, tensorflow`
 Consequently the framework has not yet initialized any parameters.
 We confirm by attempting to access the parameters below.
+:end_tab:
+
+:begin_tab:`jax`
+As mentioned previously, parameters and the network definition are decoupled
```

Comment:
"previously" is vague. Be specific where it was mentioned by using :numref:

Reply:
Done!